docs/NSR.md

```diff
@@ -347,11 +340,11 @@ The CA MUST enforce the use of Multi-Party Control for physical access to any Ro
 
 The CA SHOULD ensure passwords used as authentication credentials for accounts on CA Infrastructure, Network Equipment, or Workstations are generated and managed in accordance with NIST 800-63B Appendix A.
 
-The CA SHALL NOT require periodic password changes with a period less than two (2) years.
+The CA SHOULD NOT require periodic password changes with a period less than two (2) years.
 
-The CA MUST ensure passwords used as authentication credentials for accounts on CA Infrastructure have a minimum of twelve (12) characters.
+The CA MUST require that passwords used as authentication credentials for accounts on CA Infrastructure, Network Equipment or Workstations have a minimum of twelve (12) characters.
```

Comment:
I like ensure. In my mind, it puts the onus on the CA. Of course the CA will say "all we can do is require", but the risk for that should be on the CA. "We required it by policy but our staff willingly chose to ignore the requirement." should not be a valid excuse.

Reply:
I'm confused. If you're willing to acknowledge the necessity of 'all we can do is require', why stipulate that they do something different than that? CAs should be able to fully comply with the BRs as written.
I see 'ensure' as an unrealistic standard that is very difficult to prove in an audit unless all passwords were created and put into place by systems which log the process. In most environments at least some portion of the systems are decentralized with some degree of ad-hoc password management. 
To prove to the auditor you've 'ensured' it is 12 characters, you'd have to be able to provide proof of the creation of the password, proof of supplying it to the system and that it hadn't been changed since then. Maybe perform the whole process and record it? I mean, there are ways to accomplish this, but it seems kind of ridiculous to me, but something like that seems necessary if an auditor wants you to prove you 'ensure' this.

Reply:
Going to delete 350 and 352.